packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -641,10 +641,30 @@ contract StandardValidatorV200 is StandardValidatorBase {
     }
 
     function l1StandardBridgeVersion() public pure override returns (string memory) {
-        return "2.2.1";
+        return "2.2.2";
     }
 
     function disputeGameFactoryVersion() public pure override returns (string memory) {
         return "1.0.1";
     }
+
+    function anchorStateRegistryVersion() public pure override returns (string memory) {
+        return "2.2.2";
+    }
+
+    function delayedWETHVersion() public pure override returns (string memory) {
+        return "1.3.0";
+    }
+
+    function mipsVersion() public pure override returns (string memory) {
+        return "1.3.0";
+    }
+
+    function permissionedDisputeGameVersion() public pure override returns (string memory) {
+        return "1.4.1";
+    }
```

Comment:
Looks like we use this for both FDG and PDG games. Eventually that abstraction (assuming all game types have the same version) will break, but for now this is ok